README.md

```diff
@@ -0,0 +1,68 @@
+# Building a singular container for HPC using globus-compute
+
+## Context
+* One of the executions configurations of [globus-compute](https://www.globus.org/compute) requires a registered container which is spun up to execute the user function on the HPC.
+
+* HPCs do not run docker containers(due to security reasons as discussed [here](https://docs.vscentrum.be/software/singularity.html)) and support only an apptainer/singularity image.
+
+* Installing the apptainer setup to build the singularity image locally is not a straightforward process especially on windows and mac systems as discussed in the [documentation](https://apptainer.org/docs/admin/main/installation.html).
+
+Using this python library the user can specify their custom image specification to build an apptainer/singularity image 
+which would be used to in-turn to run their functions on globus-compute. The library registers the container and 
+returns the container id which would be used by the globus-compute executor to execute the user function.
+
+
+## Prerequisite.
+A [globus-compute-endpoint](https://globus-compute.readthedocs.io/en/latest/endpoints.html) setup on HPC cluster.
+In the following example the above endpoint would be your image builder endpoint.
+
+You will need another endpoint to run your code in your custom container. You can refer the example_config.py to setup your endpoint.
```

Comment:
Is this actually true?

Reply:
Yes I have modified the README.md to be more clear